istioctl/pkg/precheck/precheck.go

```diff
@@ -102,10 +102,10 @@ func Cmd(ctx cli.Context) *cobra.Command {
 			}
 
 			if len(outputMsgs) == 0 {
-				fmt.Println(cmd.ErrOrStderr(), color.New(color.FgGreen).Sprint("✔")+" No issues found when checking the cluster. Istio is safe to install or upgrade!\n"+
-					"  To get started, check out https://istio.io/latest/docs/setup/getting-started/")
+				_, _ = fmt.Fprintln(cmd.ErrOrStderr(), color.New(color.FgGreen).Sprint("✔")+" No issues found when checking the cluster. Istio is safe to install or upgrade!"+
```

Comment:
will this be the only place with this bug?

Reply:
will review all changes from last commit later.

Reply:
didn't find more